src/spectrecoff/Canvas.fs

```diff
@@ -0,0 +1,59 @@
+[<AutoOpen>]
+module SpectreCoff.Canvas
+
+open Spectre.Console
+
+type Width = Width of int
+[<RequireQualifiedAccess>]
+module Width =
+    let unwrap (Width w) = w
+
+type Height = Height of int
+[<RequireQualifiedAccess>]
+module Height =
+    let unwrap (Height h) = h
+
+type ColumnIndex = ColumnIndex of int
+type RowIndex = RowIndex of int
+type StartIndex = StartIndex of int
+type EndIndex = EndIndex of int
+
+type PixelPosition = int*int
+type Pixel = PixelPosition*Color
+type Pixels = 
+    | SinglePixel of PixelPosition
```

Comment:
If I read it correctly, this convenience of entering pixels like this is not possible in Spectre? If so, this is a _really_ good idea in my opinion :)

Reply:
Yes, Spectre 'just' provides the canvas. The convenience we added here is easily done by consumers, is certainly their stance., which I absolutely get in such a popular, complex library. Anything they offer, they will have to support.